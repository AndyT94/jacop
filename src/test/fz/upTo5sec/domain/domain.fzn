var 1..1: low:: output_var;
var 9..9: up:: output_var;
array [1..3] of var int: x:: output_array([1..3]) = [2,8,2];
solve  satisfy;
