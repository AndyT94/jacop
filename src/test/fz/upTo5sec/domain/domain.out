low = 1;
up = 9;
x = array1d(1..3, [2, 8, 2]);
----------